reference/activity_manager/project_completeness.rst

```diff
@@ -0,0 +1,111 @@
+What is the project completeness?
+=================================
+
+Overview
+________
+
+In this reference guide you will learn what the project completeness refers to and how we manage it in Akeneo PIM. It
+will explain our technical and structure choices.
+
+Goal
+____
+
+The Activity Manager feature is about creating projects from a products selection that gives a vision on the enrichment
+progress to contributors (users in charge of the enrichment). Each project gives to the contributor its own completeness
+on the products selection, taking into account its permissions to edit or not products, attribute groups, locales and
```

Comment:
Are we talking about the user's permissions? In that case _their permissions_, otherwise I did not understand, sorry